src/source/feeds.rst

```diff
@@ -31,6 +82,683 @@ XML file.
+
+*SHIP* means the item can be delivered to the buyer in the provided `time` and for the provided `cost`. 
+For shippingType 'SHIP' provide 'cost' in cents and 'time' in days. 'location' is ignored.
+
+*PICKUP* means the item can be picked up at the provided `location`
+For shippingType 'PICKUP' provide 'location'. Both 'cost' and 'time' are ignored.
+
+Whether shipping options are disabled/optional/mandatory for an ad is configured per category, see :ref:`category_config_v2`.
+
+======= ==============================================================
+Example .. code-block:: html 
+
+            <ad:shippingOptions>
+                <ad:shippingOption>
+                    <ad:shippingType>PICKUP</ad:shippingType>
+                    <ad:location>1097DN</ad:location>
+                </ad:shippingOption>
+            </ad:shippingOptions>
+======= ==============================================================
+
+.. index:: phoneNumber
+.. _feed_phoneNumber:
+
+phoneNumber
+"""""""""""
+
+The phone number of the seller as international phone number format, e.g.
++31207894561 or as a local phone number, e.g. 06789456612.
+
+======= =================================================
+Example .. code-block:: html
+        
+         <ad:phoneNumber>+31207894561</ad:phoneNumber>
+======= =================================================
+
+.. index:: emailAdvertiser
+.. _feed_emailAdvertiser:
+
+emailAdvertiser
+"""""""""""""""
+
+Flag which enables emails, (or the other tenant defined forms of contact) to the seller.
+The default value is false.
+
+======= =================================================
+Example .. code-block:: html
+        
+         <ad:emailAdvertiser>true</ad:emailAdvertiser>
+======= =================================================
+
+.. index:: regionId
+.. _feed_regionId:
+
+regionId
+""""""""
+
+The region in which the ad is placed. (only applicable for Kijiji Canada)
+An integer value from the region tree. Must be the id of a leaf region.
+
+Each ad belongs to one and only one region and region of an ad cannot be updated.
+This field can only be set once during creation of an ad.
+
+This field is mandatory if the `region` field of category configuration is ``MANDATORY``
+and optional if the `region` field is ``OPTIONAL``.
+This field must be omitted if the `region` field of category configuration is ``DISABLED``.
+
+Please refer to :ref:`categories` and :ref:`regions`
+
+======= =======================================
+Example .. code-block:: html
+        
+         <ad:regionId>1700274</ad:regionId>
+======= =======================================
+
+.. index:: microTip
+.. _feed_microTip:
+
+microTip
+""""""""
+
+A short freeform text with a maximum length of 18 characters, excluding any characters in ``.,/@#<>``.
+It is a feature as part of a package that sellers can purchase (currently available only for Marktplaats tenant).
+It provides extra attention on the ad in the search results.
```

Comment:
should we add that if the seller is not enabled for this feature, this input is ignored?

Reply:
updated